packages/features/ee/round-robin/roundRobinManualReassignment.ts

```diff
@@ -262,22 +266,111 @@ export const roundRobinManualReassignment = async ({
       })
     : null;
 
-  const results = await eventManager.reschedule(
+  const updateManager = await eventManager.reschedule(
     evt,
     booking.uid,
     undefined,
     hasOrganizerChanged,
     previousHostDestinationCalendar ? [previousHostDestinationCalendar] : []
   );
 
-  const newReferencesToCreate = structuredClone(results.referencesToCreate);
+  const results = updateManager.results;
+
+  const { metadata: videoMetadata, videoCallUrl: _videoCallUrl } = getVideoCallDetails({
+    results: results,
+  });
+
+  let metadata: AdditionalInformation = {};
+  metadata = videoMetadata;
+
+  if (results.length) {
+    // Handle Google Meet results
+    // We use the original booking location since the evt location changes to daily
+    if (bookingLocation === MeetLocationType) {
+      const googleMeetResult = {
+        appName: GoogleMeetMetadata.name,
+        type: "conferencing",
+        uid: results[0].uid,
+        originalEvent: results[0].originalEvent,
+      };
```

Comment:
Using the same code from handleNewBooking

Reply:
Can we reuse this code instead of copy/pasting?

Reply:
This is important logic that ideally lives in 1 place

Reply:
handleNewBooking is also doing a lot of things during rescheduling.  It's mutating a lot of variables which could be difficult to track.